src/test/java/Homework19.java

```diff
@@ -11,9 +11,10 @@ public class Homework19 extends BaseTest {
 
     @Test
     public void DeletePlaylistTest() throws InterruptedException {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example public void **deletePlaylistTest()**